src/backend/backend.did

```diff
@@ -10,6 +10,7 @@ type AddUserCredentialRequest = record {
   current_user_version : opt nat64;
   credential_spec : CredentialSpec;
 };
```

Comment:
We wish to be able to make features either fully enabled, fully disabled, or enabled for read methods only.  This type is used to represent which of these states a feature is in.